{
  "debianbug": 850716,
  "description": "XML External Entity (XXE) vulnerability in PySAML2 4.4.0 and earlier allows remote attackers to read arbitrary files via a crafted SAML XML request or response.",
  "releases": {
    "bullseye": {
      "fixed_version": "3.0.0-5",
      "repositories": {
        "bullseye": "4.5.0-8"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "buster": {
      "fixed_version": "3.0.0-5",
      "repositories": {
        "buster": "4.5.0-4+deb10u1",
        "buster-security": "4.5.0-4+deb10u1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "jessie": {
      "fixed_version": "2.0.0-1+deb8u1",
      "repositories": {
        "jessie": "2.0.0-1+deb8u1",
        "jessie-security": "2.0.0-1+deb8u3"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "sid": {
      "fixed_version": "3.0.0-5",
      "repositories": {
        "sid": "4.5.0-8"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "stretch": {
      "fixed_version": "3.0.0-5",
      "repositories": {
        "stretch": "3.0.0-5",
        "stretch-security": "3.0.0-5+deb9u1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    }
  },
  "scope": "local"
}